website/metadata.js

```diff
@@ -64,6 +64,15 @@ module.exports = {
       },
       "description": "Take Vector from zero to production in under 10 minutes.",
       "guides": [
+        {
```

Comment:
Ran `make generate` because that's what test failure said.

Reply:
Could you rebase with `master`? This should be resolved.

Reply:
Rebased fixed it.